syscall_good.go

```diff
@@ -1,8 +1,5 @@
-// +build !plan9,!windows
-// +build !js !wasm
-
 package sftp
 
-import "syscall"
+import sshfx "github.com/pkg/sftp/internal/encoding/ssh/filexfer"
 
-const S_IFMT = syscall.S_IFMT
+const S_IFMT = sshfx.ModeType
```

Comment:
There’s now no need to have two copies of this an only pull it in based on GOOS/GOARCH pairs that have a bad value.

We do export the value though, and it was intended to be used, so we’re keeping it exported. At any possible v2, we can remove it in light of using the `ssh/filexfer.ModeType` value along with its brethren directly.